TombEngine/Renderer/RendererDrawMenu.cpp

```diff
@@ -129,6 +128,9 @@ namespace TEN::Renderer
 		auto screenResolution = g_Configuration.SupportedScreenResolutions[g_Gui.GetCurrentSettings().SelectedScreenResolution];
 		sprintf(stringBuffer, "%d x %d", screenResolution.x, screenResolution.y);
 
+		char soundDeviceStringBuffer[255] = {};
+		sprintf(soundDeviceStringBuffer, "%s", g_Configuration.SupportedSoundDevices[g_Gui.GetCurrentSettings().SelectedSoundDevice].Name.c_str());
```

Comment:
Sound device name may contain non-Latin characters (e.g. default sound device on non-English systems). Maybe a good idea is to use Unicode here?